src/deprecation.js

```diff
@@ -918,6 +918,24 @@ Object.defineProperty(core.utils, '_saidHello', {
 });
 
 /**
+ * Adds hooks for finding and uploading items.
+ *
+ * @param {Function} [addHook] - Function call that takes two parameters: `item:*, queue:Array`
```

Comment:
I like to remove parameters from the deprecation, so that people don't use it. Also this needs a proper JSDoc:
```js
/**
 * @method
 * @name PIXI.prepare.BasePrepare#register
 * @see PIXI.prepare.BasePrepare#registerFindHook
 * @deprecated since version 4.4.2
 */
```

You can test this by running `npm run docs` and opening **docs/index.html**